scripts/claude_mcp.sh

```
@@ -1,34 +1,16 @@
 #!/usr/bin/env bash
 
-TEST_MODE=false
-if [ "$1" == "--test" ]; then
-    TEST_MODE=true
-fi
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+MCP_LAUNCHER_PATH="$0"
```

Shellcheck (0.11.0)

[warning] MCP_LAUNCHER_PATH appears unused. Verify use (or export if used externally).

(SC2034)